<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Switches_box</class>
 <widget class="QDialog" name="Switches_box">
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>D1</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="cb_D1_1">
          <property name="text">
           <string>D1.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D1_2">
          <property name="text">
           <string>D1.2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D1_3">
          <property name="text">
           <string>D1.3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D1_4">
          <property name="text">
           <string>D1.4</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>D4</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="cb_D4_1">
          <property name="text">
           <string>D4.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D4_2">
          <property name="text">
           <string>D4.2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D4_3">
          <property name="text">
           <string>D4.3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D4_4">
          <property name="text">
           <string>D4.4</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>D5</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="cb_D5_1">
          <property name="text">
           <string>D5.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D5_2">
          <property name="text">
           <string>D5.2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D5_3">
          <property name="text">
           <string>D5.3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D5_4">
          <property name="text">
           <string>D5.4</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>D6</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="cb_D6_1">
          <property name="text">
           <string>D6.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D6_2">
          <property name="text">
           <string>D6.2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D6_3">
          <property name="text">
           <string>D6.3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D6_4">
          <property name="text">
           <string>D6.4</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>D15</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="cb_D15_1">
          <property name="text">
           <string>D15.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D15_2">
          <property name="text">
           <string>D15.2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D15_3">
          <property name="text">
           <string>D15.3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D15_4">
          <property name="text">
           <string>D15.4</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>D16</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QCheckBox" name="cb_D16_1">
          <property name="text">
           <string>D16.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D16_2">
          <property name="text">
           <string>D16.2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D16_3">
          <property name="text">
           <string>D16.3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D16_4">
          <property name="text">
           <string>D16.4</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="title">
        <string>D27</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QCheckBox" name="cb_D27_1">
          <property name="text">
           <string>D27.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D27_2">
          <property name="text">
           <string>D27.2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D27_3">
          <property name="text">
           <string>D27.3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_D27_4">
          <property name="text">
           <string>D27.4</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_D1_1</tabstop>
  <tabstop>cb_D1_2</tabstop>
  <tabstop>cb_D1_3</tabstop>
  <tabstop>cb_D1_4</tabstop>
  <tabstop>cb_D4_1</tabstop>
  <tabstop>cb_D4_2</tabstop>
  <tabstop>cb_D4_3</tabstop>
  <tabstop>cb_D4_4</tabstop>
  <tabstop>cb_D5_1</tabstop>
  <tabstop>cb_D5_2</tabstop>
  <tabstop>cb_D5_3</tabstop>
  <tabstop>cb_D5_4</tabstop>
  <tabstop>cb_D6_1</tabstop>
  <tabstop>cb_D6_2</tabstop>
  <tabstop>cb_D6_3</tabstop>
  <tabstop>cb_D6_4</tabstop>
  <tabstop>cb_D15_1</tabstop>
  <tabstop>cb_D15_2</tabstop>
  <tabstop>cb_D15_3</tabstop>
  <tabstop>cb_D15_4</tabstop>
  <tabstop>cb_D16_1</tabstop>
  <tabstop>cb_D16_2</tabstop>
  <tabstop>cb_D16_3</tabstop>
  <tabstop>cb_D16_4</tabstop>
  <tabstop>cb_D27_1</tabstop>
  <tabstop>cb_D27_2</tabstop>
  <tabstop>cb_D27_3</tabstop>
  <tabstop>cb_D27_4</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Switches_box</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Switches_box</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
